vnext/PropertySheets/React.Cpp.props

```diff
@@ -43,7 +42,7 @@
     </Link>
   </ItemDefinitionGroup>
 
-  <ItemDefinitionGroup Condition="'$(Platform)' == 'x64' OR '$(Platform)' == 'x86'">
+  <ItemDefinitionGroup Condition="'$(Platform)' == 'x64' OR '$(Platform)' == 'x86' OR '$(Platform)' == 'Win32'">
```

Comment:
>'x86' OR '$(Platform)' == 'Win32' [](start = 77, length = 33)

I'm confused - what is the name of the platform now, 'x86' or 'Win32'?

Reply:
For C++ projects, `Win32` was the value used to refer to the  32 bit `ix86` architecture.
After `x64` was introduced, the MSVC team realized `x86` was a better alternativer, but for some workflows (like Visual Studio IDE project loading), `Win32` is still the value used by the build system.

In short, building the same VCXPROJ with either `/p:Platform=Win32` or `/p:Platform=x86` yields the same binary artifacts, but the paths for `$(IntDir)` and `$(OutDir)` may differ.

These predicates ensure that whether building within the IDE or from command line, we get the same behavior.

Reply:
Pardon the somewhat facetious question. I understand what that means, I'm questioning the value of a choice without distinction. Why not universally refer to the platform as Win32?

---
In reply to: [296922183](https://github.com/microsoft/react-native-windows/pull/2664#discussion_r296922183) [](ancestors = 296922183)

Reply:
Discussed offline.
$(Platform) can be set as either `Win32` or `x86`. The output paths will contain `x86` irregardless.